scripts/west_commands/runners/mdb.py

```diff
@@ -18,6 +20,29 @@
 def simulation_run(mdb_runner):
     return mdb_runner.nsim_args != ''
 
+def get_cld_pid(mdb_process):
```

Comment:
IMHO would be good to add some background info in the commit message on why that PID mess is needed.
I.e. that MDB spawns that CLD process which handles real work and that just killing MDB doesn't affect CLD process, thus we need to reach it directly by its PID.

Reply:
We have comment with some info in the place were we call get_cld_pid. However I've added more detailed description and move it to the place where we define get_cld_pid.